Description: fix for dh_autoreconf
 This patch fix for dh_autoreconf and also do help for bug #931163
Forwarded: not-needed
 .
Author: xiao sheng wen <atzlinux@sina.com>

---

Bug-Debian: https://bugs.debian.org/931163
Last-Update: 2021-01-19

--- tnftp-20200705.orig/configure.ac
+++ tnftp-20200705/configure.ac
@@ -89,6 +89,7 @@ AH_TEMPLATE([WITH_SSL],
 #
 AC_PROG_CC()
 AC_PROG_AWK()
+AM_PROG_AR()
 AC_PROG_LIBTOOL()
 
 #
--- tnftp-20200705.orig/libnetbsd/Makefile.am
+++ tnftp-20200705/libnetbsd/Makefile.am
@@ -4,7 +4,7 @@ noinst_LTLIBRARIES = libnetbsd.la
 
 libnetbsd_la_SOURCES = libnetbsd.c
 
-CPPFLAGS = \
+AM_CPPFLAGS = \
 	-I$(srcdir) \
 	-I$(top_srcdir) \
 	-I$(top_builddir)
